lib/Wikia/tests/Util/SamplerProxyTest.php

```diff
@@ -0,0 +1,312 @@
+<?php
+
+use Wikia\Util\SamplerProxy;
+
+class SamplerProxyTest extends \PHPUnit_Framework_TestCase {
+
+	/** @var \PHPUnit_Framework_MockObject_MockObject */
+	private $originalMock;
+
+	/** @var \PHPUnit_Framework_MockObject_MockObject */
+	private $alternateMock;
+
+	function setUp() {
+		parent::setUp();
+
+		$this->originalMock = $this->getMockBuilder( OriginalPopo::class )->setMethods( [
+			'methodToSample',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "methodToSample" 11 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)